lib/signaling/v2/peerconnection.js

```diff
@@ -101,12 +101,16 @@ class PeerConnectionV2 extends StateMachine {
     const peerConnection = new RTCPeerConnection(configuration, options.chromeSpecificConstraints);
     const sdpFormat = getSdpFormat(options.sdpSemantics);
 
-    const localMediaStream = isFirefox && RTCPeerConnection.prototype.addTransceiver
+    const localMediaStream = sdpFormat === 'unified' && RTCPeerConnection.prototype.addTransceiver
```

Comment:
We do this check a couple times—can we have a flag, `isUnified`, instead? IMO, it would be more readable and less error-prone in the event the string changes.